Dockerfile

```diff
@@ -1,7 +1,22 @@
-FROM start9/hello-world
+FROM ghcr.io/itchysats/itchysats/taker:0.6.1 as builder
 
-RUN apk add --no-cache curl && \
-    rm -f /var/cache/apk/*
+## TODO: It would be simpler to use our container, but it is based on distroless - which is great because it's slim, but it does not have any sheel support
```

Comment:
An alternative approach could be to have 2 containers? One for the health check and the otherone from us.

Reply:
You mean a healthcheck container that interacts with the `itchysats` container just for querying the endpoints? 

🤔 That's an interesting idea I had not considered; I am not sure how we would set this up. The wrapper does not have control of the `docker run` command, but only `Dockerfile` and `entrypoint.sh`, the latter being supplied to the container to configure the startup of the container. I don't think I can easily start two containers within the constraints of this setup but maybe I'm missing something.

Reply:
ah ok, I kinda expected it to have some sort of docker-compose file similar to umbrel. Thanks for the clarification.